test/fixtures/user_fixtures.js

```diff
@@ -30,6 +179,64 @@ function merchantDetailsFixture () {
   }
 }
 
+const validServiceRole = (opts = {}) => {
+  return {
+    service: validService(opts.service),
+    role: validRole(opts.role)
+  }
+}
+
+const validService = (opts = {}) => {
```

Comment:
This actual functionality of this might be clearer with a more specific name

```suggestion
const buildServiceWithDefaults = (opts = {}) => {
```

Then it might read 
```js
{ 
  service: buildServiceWithDefaults(opts.service) 
}
```

This name is just a suggestion however `validService` sounds like a static object.

This goes for all `valid` objects.

Reply:
Done